import_export_extensions/resources.py

```diff
@@ -141,8 +195,39 @@ def import_row(
                 else TaskState.PARSING.name
             ),
         )
+        if not force_import:
+            return imported_row
+        if (
+            imported_row.import_type == results.RowResult.IMPORT_TYPE_ERROR
+            or imported_row.import_type == results.RowResult.IMPORT_TYPE_INVALID
+        ):
+            imported_row.diff = []
+            for field in self.get_fields():
+                imported_row.diff.append(row.get(field.column_name, ""))
+
+            imported_row.non_field_skipped_errors.extend(
+                imported_row.errors,
+            )
+            if imported_row.validation_error is not None:
+                imported_row.field_skipped_errors.update(
+                    **imported_row.validation_error.error_dict,
+                )
+            imported_row.errors = []
+            imported_row.validation_error = None
+
+            imported_row.import_type = results.RowResult.IMPORT_TYPE_SKIP
         return imported_row
 
+    @classmethod
+    def get_row_result_class(self):
+        """Return custom row result class."""
+        return SkippedErrorsRowResult
```

Comment:
Name of class is kinda confusing, because it makes you expect that result is skipped

Reply:
Maybe `CeleryErrorsResult`?

Reply:
I think `CeleryErrorsResult` doesn't reflect the information that the class is containing skipped errors. What about `SkippedRow` and `ResultWithSkippedRows`?

Reply:
Hm, well that's better)